src/Menus.h

```diff
@@ -113,8 +114,9 @@ class AUDACITY_DLL_API MenuManager final
    void TellUserWhyDisallowed(const TranslatableString & Name, CommandFlag flagsGot,
       CommandFlag flagsRequired);
 
-   void OnUndoRedo( wxCommandEvent &evt );
+   void OnUndoRedo(struct UndoRedoMessage);
```

Comment:
Same a in Lyrics.h, struct keyword is not needed.

Reply:
Also incorrect for the same reason.